pkgs/by-name/ho/hof/package.nix

```diff
@@ -0,0 +1,35 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+}:
+
+let
+  pname = "hof";
+  version = "0.6.9-beta.1";
+in
+buildGoModule {
+  inherit pname version;
+
+  src = fetchFromGitHub {
+    owner  = "hofstadter-io";
+    repo   = "hof";
+    rev    = "v${version}";
+    hash   = "sha256-4yVP6DRHrsp52VxBhr7qppPhInYEsvPbIfxxQcRwHTw=";
+  };
+
+  vendorHash = "sha256-cDUcYwcxPn+9TEP5lhVJXofijCZX94Is+Qt41PqUgjI=";
+
+  subPackages = [ ".hof/shadow/cli/cmd/hof/main.go" ];
```

Comment:
should this be cmd/hof ?

https://github.com/hofstadter-io/hof/blob/_dev/.github/workflows/hof.yml#L69C25-L69C34

Reply:
good catch, thanks!